### ISSUE TITLE
About the HotpotQA dev set

### ISSUE THREAD

**Issue by kaigexie** [allenai/modularqa#2]
Hi, thanks for the nice paper!

I have three questions regarding the HotpotQA dev set:

(1) In hotpot_dev.txt of "NextGen Training Data" (modularqa_nextgen_train.zip), there are 676 different questions. Could you explain how you get this subset from the overall 7405 questions of the hotpot dev set?
(2) In hotpot_dev.txt, for each question there are several decomposition chains. Are they pre-sorted by the scores given by the chain scorer? In other words, how do I know which one is the one that has the highest score?
(3) In dev.json of "HotpotQA" (hotpot_subset.zip), there are 1481 different questions. Same as (1), could you explain how you get this subset from the overall 7405 questions of the hotpot dev set? I notice that in the paper you mention the original hotpotQA dev set is split into 1481 for dev & 5924 for test. Are they randomly split?

Looking forward to your reply! :)

**Comment by tusharkhot** [comment]
(2) These chains are used for training/evaluation of the chain scorer. So we don't really have scores on these chains from the scorer. They could be generated using the chain scorer model.
(3) Yes you are right. They were randomly split to create our own dev-test split. Note we didn't have the answers of the original hotpot test set. 
(1) Given the answer to (3), the specific examples you see in NextGen training data are the ones where we were successfully able to find the chains that lead to the gold answer.
